main.go

```diff
@@ -81,6 +81,31 @@ func main() {
 		exit = true
 	}
 
+	if opts.Edit != nil {
+		editor := os.Getenv("EDITOR")
+		editors := []string{editor, "nano", "vim", "vi"}
+		for _, ed := range editors {
+			editor, err = exec.LookPath(ed)
+			if err == nil {
+				break
+			}
+		}
+		if editor != "" {
+			configDir, _ := os.UserConfigDir()
+			if *opts.Edit == "" {
+				*opts.Edit = "dxhd.sh"
+			}
+			path := filepath.Join(configDir, "dxhd", *opts.Edit)
+			err = syscall.Exec(editor, []string{editor, path}, os.Environ())
+			if err != nil {
+				logger.L().WithField("Error", err).Fatal("Error while executing editor.")
+			}
+		} else {
+			logger.L().Fatal("Cannot find a suitable editor to open. Please set one in $EDTIOR")
```

Comment:
Follow the style of other error messages in dxhd (found in parser.go), and start the error message without "error" word", and do not end with period.